code/modules/clothing/suits/marine_armor/intel.dm

```diff
@@ -84,7 +84,7 @@
 	// Only equip if uniform doesn't already have a utility accessory slot equipped
 	var/obj/item/clothing/under/uniform = user.w_uniform
 	var/accessory = locate(/obj/item/clothing/accessory/storage) in uniform.accessories
-	if(accessory)
+	if(accessory && !istype(accessory,/obj/item/clothing/accessory/storage/ifak))
 		to_chat(user, SPAN_WARNING("[src] can't be worn with [accessory]."))
 		return FALSE
```

Comment:
I'm not sure about this part. It has a pretty big side effect of giving IO players an extra 5ish slots of inventory space for free, since any medical supplies they want to bring could go into the IFAK and free up a pouch slot/backpack space.

Reply:
I intended this to be ubiquitous, for _everyone_ that deploys to the ground to use it if they don't take another webbing. Based off of player feedback the restriction on webbing for the XM4 was rather harsh, this would bring it closer to balance without giving too much of an advantage. However, if you feel this is too much of an advantage for them I can remove it.

Reply:
Discussed this a bit in maint chat, and I think the IFAK being the only exception to the 'no webbings' rule would mostly be confusing, plus the storage space thing, so I'd say to leave it out for now yeah.

Reply:
It has been removed.